Fix linting thrown from default template when enabling prettier option, container centering when enabling tailwind option

liam-potter opened the pull request:
This PR should resolve the errors that are thrown by the default template when users enable the prettier option during project creation.

aldarund commented:
@liam-potter Nice catch :) What about prettier errors with all other templates e.g. vuetify, buefy etc. Did u checked it?

liam-potter commented:
@aldarund The only other framework template with CSS changes required for prettier was Vuetify - I have now updated the required files too.

I have also fixed https://github.com/nuxt-community/create-nuxt-app/issues/97 in this PR.

liam-potter commented:
https://github.com/nuxt/nuxt.js/issues/3933 will also be resolved by this (related to https://github.com/nuxt-community/create-nuxt-app/issues/97).
